src/main/java/org/folio/dew/batch/acquisitions/services/ConfigurationService.java

```diff
@@ -53,4 +50,12 @@ public String getAddressConfig(UUID shipToConfigId) {
       return "";
     }
   }
+
+  private ModelConfiguration getConfigById(String configId) {
+    try {
+      return configurationClient.getConfigById(configId);
+    } catch (NotFoundException e) {
+      throw new NotFoundException("Configuration entry not found for id: " + configId, e);
```

Comment:
this private method can be deleted. Nowwe will throw new NotFoundException that will contain NotFoundException in the cause. SO we can improve this existing behaviour by deleting this method